test/Conversion/TorchToTosa/constant_pad_nd.mlir

```diff
@@ -0,0 +1,37 @@
+// RUN: torch-mlir-opt <%s -convert-torch-to-tosa -split-input-file -verify-diagnostics | FileCheck %s
```

Comment:
The preference in torch-mlir is to not add unit test cases for straightforward expansions like this, we rely on the e2e testing. You have enabled e2e testing above, so you don't need this file.
`DON'T use a unit test if your lowering pattern could be described as a trivial "macro expansion" of one op into another op or set of ops. That is, if you feel like your unit test is just rewriting b.create<...>(...) into CHECK: ... then it is probably not a useful unit test.`
https://github.com/llvm/torch-mlir/blob/main/docs/architecture.md#dos-and-donts-for-unit-vs-end-to-end-testing

Reply:
@eric-k256 Thanks for your review. I've removed the unit test test/Conversion/TorchToTosa/constant_pad_nd.mlir.